records/SMP_adding_translation.md

```diff
@@ -0,0 +1,6 @@
+# Process of adding a new string using the Translation component in sm_platform
+When changes are made with a string that uses the Translate component or translate() function in a PR, file a ticket or a follow up PR to ensure that the correspoding Chinese and Russian language files are updated as well.
+The steps to update are as follows:
+* Run `npm run extract` to make `en.json` file updated
+* Manually updates `ru.json` file, ask Sasha for language help.
```

Comment:
I've done this once, and I vaguely recall it involved locating the added IDs from `en.json` and adding corresponding entries to `ru.json`.  If that's the process, it would be good to have some additional detail on it for anyone who hasn't done this before at all.